todo/Services/IAccountService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Todo.Models;
+using Todo.DTOs;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Todo.Services
+{
+    public interface IAccountService
+    {
+        public Task<ActionResult<SignUpDTO>> SignUpAsync(SignUpDTO model);
+        public Task<string> SignInAsync(SignInDTO model);
```

Comment:
```suggestion
        Task<ActionResult<SignUpDTO>> SignUpAsync(SignUpDTO model);
        Task<string> SignInAsync(SignInDTO model);
```
interface thì k cần cái này nhé